[BUG]company_list接続失敗時のエラーログが正常に出力されない

## 対象
:sob: その他

## バグの説明
company_list のhttp get に失敗した際のエラーログが正常に出力されない。

## 再現方法
以下のようなログが出力されてしまっている状態。

```    
[ERROR] 'Logger' object has no attribute 'err' [in /app/ibet-Wallet-API/app/api/v2/token.py:442]
```

## 正しい挙動（想定）
```
LOG.err("Failed to get company list: ", err)
```
というような実装になってしまっている箇所が存在する。LOG.errorが本来は正しい。

原因の切り分けのために、エラーの詳細も出力したほうが良いと考えられることから、LOG.exception でログ出力をする方が良いと思われる。

--- a/app/model/token.py
+++ b/app/model/token.py
@@ -120,7 +120,7 @@ class BondToken(TokenBase):
             try:
                 company_list = requests.get(config.COMPANY_LIST_URL, timeout=config.REQUEST_TIMEOUT).json()
             except Exception as err:
-                LOG.err("Failed to get company list: ", err)
+                LOG.exception(f"Failed to get company list: {err}")
 
         # Contractから情報を取得する
         name = TokenContract.functions.name().call()
@@ -309,7 +309,7 @@ class ShareToken(TokenBase):
             try:
                 company_list = requests.get(config.COMPANY_LIST_URL, timeout=config.REQUEST_TIMEOUT).json()
             except Exception as err:
-                LOG.err("Failed to get company list: ", err)
+                LOG.exception(f"Failed to get company list: {err}")
 
         owner_address = TokenContract.functions.owner().call()
         name = TokenContract.functions.name().call()
@@ -436,7 +436,7 @@ class MembershipToken(TokenBase):
             try:
                 company_list = requests.get(config.COMPANY_LIST_URL, timeout=config.REQUEST_TIMEOUT).json()
             except Exception as err:
-                LOG.err("Failed to get company list: ", err)
+                LOG.exception(f"Failed to get company list: {err}")
 
         # Token-Contractから情報を取得する
         name = TokenContract.functions.name().call()
@@ -559,7 +559,7 @@ class CouponToken(TokenBase):
             try:
                 company_list = requests.get(config.COMPANY_LIST_URL, timeout=config.REQUEST_TIMEOUT).json()
             except Exception as err:
-                LOG.err("Failed to get company list: ", err)
+                LOG.exception(f"Failed to get company list: {err}")
 
         # Token-Contractから情報を取得する
         name = TokenContract.functions.name().call()

